requirements.txt

```diff
@@ -1,8 +1,5 @@
-tsdat == 0.2.10
-netcdf4
-xarray
-numpy
-matplotlib
+tsdat >0.4.3, <=0.5.0
```

Comment:
Let's update this to work with tsdat 0.5.0 (VAP additions, changed DataConverters)